dockers/base-cuda/Dockerfile

```diff
@@ -22,8 +22,11 @@
 ARG CUDNN_VERSION=7
 ARG CUDA_VERSION=10.1
 
-FROM nvidia/cuda:${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel
-# FROM nvidia/cuda:${CUDA_VERSION}-devel
+# FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu20.04
+# FROM nvidia/cuda:${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel-ubuntu18.04
+FROM nvidia/cuda:${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel-ubuntu16.04
```

Comment:
this is bad, this is too. old OS version, this is the almost unsupported OS (ends in April 2021)